talk/expert.tex

```diff
@@ -403,11 +403,45 @@
       struct hasFoo<T, decltype(std::declval<T>().foo())>
         : std::true_type {};
 
-      std::cout << hasFoo<MyType>::value << std::endl;
+      struct A{}; struct B{void foo();}; struct C{int foo();};
+      static_assert(!hasFoo<A>::value, "A has no Foo()");
+      static_assert(hasFoo<B>::value, "B has Foo()");
+      // static_assert(hasFoo<C>::value, "C has Foo()"); fails assertion!
```

Comment:
Nice but does not fit at all in the slide ! Maybe just add a `\small` before the code section